lib/menu.dart

```diff
@@ -0,0 +1,92 @@
+import 'package:flutter/material.dart';
+
+const drawerMenuKey = Key("drawer_menu");
+const todoTileKey = Key("todo_tile");
+const tourTileKey = Key("tour_tile");
+const settingsTileKey = Key("settings_tile");
+
+const closeMenuKey = Key("close_key_icon");
+
+class DrawerMenu extends StatelessWidget {
+  const DrawerMenu({super.key});
+
+  @override
+  Widget build(BuildContext context) {
```

Comment:
not a _single_ comment in this file. 😢 
you might think that the code is "self-documenting" and doesn't need comments. 
but it really isn't. if you were a complete beginner reading this you wouldn't have a _clue_!

Also, formatting/indentation is off. 🤷‍♂️ 
Is there no `linting` applied to the code?

Reply:
Addressed in https://github.com/dwyl/app/pull/318